README.md

```diff
@@ -1,56 +1,46 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# React-Todo-App
 
-> Description the project.
+> A React based Todo-App based on the ReactJS [tutorial](https://ibaslogic.com/react-tutorial-for-beginners/)
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML, CSS, JavaScript
+- React JS
 
-## Live Demo (if available)
+## Live Demo 
 
-[Live Demo Link](https://livedemo.com)
+in progess :smile: but try [this](https://Efiamotu-1.github.io/react-todo-app)
 
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
 
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
-
-### Setup
-
 ### Install
-
+Have NodeJS installed and follow the next steps
 ### Usage
-
+- `npm i` for installing dev dependecies
+- `npm run build` or `npm start` to see the project
 ### Run tests
-
+- `npm test`
 ### Deployment
-
+- `npm run deploy`
 
 
 ## Authors
 
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+👤 **Habeeb Musa Efiamotu**
 
-👤 **Author2**
+- GitHub: [@githubhandle](https://github.com/Efiamotu-1)
+- Twitter: [@twitterhandle](https://twitter.com/EFYAMOTU)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/musa-habeeb/)
```

Comment:
Please be kind to update your social handles

![Screen Shot 2022-09-16 at 09 28 03](https://user-images.githubusercontent.com/99816838/190675373-cea74e10-6576-48f9-82a6-7e406256c095.png)